docs/pages/machine-id/reference/cli.mdx

```diff
@@ -0,0 +1,72 @@
+```
+</Details>
+
+| Flag                 | Description                                                                                    |
+|----------------------|------------------------------------------------------------------------------------------------|
+| `-d/--debug`         | Enable verbose logging to stderr.                                                              |
+| `-c/--config`        | Path to a configuration file. Defaults to `/etc/tbot.yaml` if unspecified.                     |
+| `-a/--auth-server`   | Address of the Teleport Auth Server (On-Prem installs) or Proxy Server (Cloud installs).       |
+| `--token`            | A bot join token, if attempting to onboard a new bot; used on first connect.                   |
+| `--ca-pin`           | CA pin to validate the auth server.                                                            |
+| `--data-dir`         | Directory to store internal bot data. Access should be limited. See `tbot init` for details.   |
+| `--destination-dir`  | Directory to write short-lived machine certificates.                                           |
+| `--certificate-ttl`  | TTL of short-lived machine certificates.                                                       |
+| `--renewal-interval` | Interval at which short-lived certificates are renewed; must be less than the certificate TTL. |
+| `--join-method`      | Method to use to join the cluster, can be `token` or `iam`.                                    |
+| `--one-shot`         | If set, quit after the first renewal.                                                          |
+
+## `tbot init`
+
+Initialize a certificate destination directory for writes from a separate bot
+user. Useful when running the bot under an isolated user account.
+
+```code
+$ tbot init \
+    --destination-dir=/opt/machine-id \
+    --bot-user=ubuntu \
+    --owner=ubuntu:ubuntu \
+    --reader-user=jenkins
+```
+
+| Flag                 | Description                                                                                           |
+|----------------------|-------------------------------------------------------------------------------------------------------|
+| `-d/--debug`         | Enable verbose logging to stderr.                                                                     |
+| `-c/--config`        | Path to a configuration file. Defaults to `/etc/tbot.yaml` if unspecified.                            |
+| `--destination-dir`  | Directory to write short-lived machine certificates to.                                               |
+| `--owner`            | Defines Linux user:group owner of `--destination-dir`. Defaults to `nobody:nobody` if unspecified.    |
+| `--bot-user`         | When using ACLs, defines Linux user that can write short-lived certificates to `--destination-dir`.   |
+| `--reader-user`      | When using ACLs, defines Linux user that will read short-lived certificates from `--destination-dir`. |
```

Comment:
Tim: For these three (`--owner`, `--bot-user`, and `--reader-user`), can you provide some more details on how they should be used.

For example, one thing that is not clear to me is what flags trigger the use of Linux ACLs?

Reply:
It will try to use ACLs if both `--bot-user` and `--reader-user` are set (and by default produces a loud warning if one or both are missing). I guess importantly, that's only on Linux, ACLs are never tried on macOS / Windows.

`init` tries to infer the owner automatically. That help message is outdated ([oops](https://github.com/gravitational/teleport/pull/11158)) as the default owner is now the bot user, but you can pass in `--owner` to use any arbitrary user:group. It'll try to fix ownership if possible (i.e. running `tbot init` as root) regardless of whether or not ACLs are in use.

Reply:
For `--owner` do you mean bot user or current Linux user that is running `tbot init`?